README.md

```diff
@@ -1,9 +1,41 @@
 # Snowflake User Terraform Module
 
-Terraform module which creates user resources on Snowflake.
-
-## Examples
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+Manage Snowflake users using Terraform.
+
+This module exposes all of the available configuration for a Snowflake user and passes them to the Snowflake provider resource. You can also supply functional roles to assign to the user during creation.
+
+## Usage
+
```

Comment:
Would it be good to add a description for what this will create?

Reply:
I've updated the README to specify what resources are being deployed.